inviteme/main.go

```diff
@@ -0,0 +1,104 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/aws/aws-lambda-go/events"
+	"github.com/aws/aws-lambda-go/lambda"
+	"github.com/gorilla/schema"
+	"net/http"
+	"net/url"
+)
+
+const (
+	inviteLink    string = "https://jdsb-test-ground-slack-invite.herokuapp.com/"
+	imageLink     string = "http://giphygifs.s3.amazonaws.com/media/5q7XU3MurT0pW/giphy.gif"
+	imageText     string = "I came in like a wrecking ball"
+	imageAltText  string = "Picture of a cat wrecking neatly stacked books"
+	linkMsg       string = "Click on the link below to get an invite to *jdsb-wrecking-ball* workspace: "
+	myJdsbSlackID        = "UGAA0BJ4R"
```

Comment:
It doesn't look like this converted to your username in your test unless you blocked it out on purpose?

Reply:
You can keep it this way for now. I'll add a way to change environments through the registry.

Reply:
Added an issue for this: #34